dll/win32/kernel32/client/session.c

```diff
@@ -13,31 +16,141 @@
 #define NDEBUG
 #include <debug.h>
 
+/* GLOBALS ********************************************************************/
+
+PBASE_STATIC_SERVER_DATA BaseStaticServerData;
+
+#define LMEM_FIXED 0x0
+
+#define ERROR_INVALID_PARAMETER 0x57
+#define ERROR_NOT_ENOUGH_MEMORY 0x8
+
 /* FUNCTIONS ******************************************************************/
 
 /*
- * @unimplemented
+ * @implemented
  */
 DWORD
 WINAPI
 DosPathToSessionPathW(IN DWORD SessionID,
                       IN LPWSTR InPath,
                       OUT LPWSTR *OutPath)
 {
-    UNIMPLEMENTED;
+    DWORD LastError = 0;
```

Comment:
```suggestion
    DWORD LastError = ERROR_SUCCESS;
```
Use macro instead of hard-code.